test/test_oa_create_request_body.py

```diff
@@ -0,0 +1,505 @@
+from zalo_sdk.oa.Client import Client
+from zalo_sdk.oa.ZaloMessage import *
+
+from . import ZaloSendMessage
+
+
+class ZaloSendRequestBody(ZaloSendMessage):
+
+    def test_send_text_message_body(self):
+        recipient = ZaloRecipient(user_id=self.user_id)
+        message_body = ZaloMessageBody(text="Hello, world!")
+
+        client = Client(self.app_id, self.secret_key, self.access_token, self.refresh_token)
+
+        # Call the send_message function and assert the response
+        body = client.create_request_body(recipient=recipient, body=message_body)
+
+        data = {
```

Comment:
For all the tests that we comparing a generated value to a known value, we should name the variables as `generated_$name` for generated value, and `expected_$name` for the known value. For this case, we call them `generated_body` and `expected_body`.

Please change rename the variables accordingly :)